app/controllers/preferences_controller.rb

```diff
@@ -5,4 +5,24 @@ def index
     @preferences = current_user.preferences
     @pagy, @records = pagy(@preferences)
   end
+
+  def new
+    @preference = current_user.preferences.build
```

Comment:
I think is enough with

```suggestion
    @preference = Preferences.new
```